tripleblindmarket/covid-safe-paths
# (Fix) #672 - remove confusing error logging in Google import

What's done:
- Remove excessive logging caused confusion in ticket #672. The import is actually completed successfully
- Add an error message in case of denied access if Take out is opened from other than Google Drive folder

#### Linked issues:

Fixes #672 

#### Screenshots:

<img width="320" alt="Screen Shot 2020-04-27 at 11 06 31 PM" src="https://user-images.githubusercontent.com/51369852/80442921-fce06080-88db-11ea-9cff-ede4e5d10922.png">


#### How to test

To verify import is successful:
1. Upload google takeout file
2. Go to import screen
3. Click "import locations"
4. You should see the success message and no errors in the console

To check out access denied error message (Android only):
1. Download Take out file in browser from google drive mobile site
2. Go to import screen
3. Click "import locations"
4. Select Take out file from Downloads folder
